frontend/src/Judge/pages/Vote.js

```diff
@@ -99,12 +104,14 @@ class Vote extends Component {
 
   componentWillUnmount () {
     document.removeEventListener('keydown', this.handleKeyInput)
+    document.body.style = 'background: white'
   }
 
   componentDidMount () {
     this.props.fetchSubmissions()
     this.props.fetchVotes()
     document.addEventListener('keydown', this.handleKeyInput)
+    document.body.style = 'background: #777777'
```

Comment:
Not sure if I like this, feels a bit hacky.

Reply:
I think this is the only way to handle it and account for overscrolling it.